tests/test_redcap_interface.py

```diff
@@ -45,3 +46,47 @@ def test_cli_writes_file(temp_out_dir, set_top_level_dir):
 
     expected_file = temp_out_dir / "processed" / "masterfile_2021-22.xlsx"
     assert expected_file.exists()
+
+
+def test_school_id_aliases(temp_out_dir, set_top_level_dir):
+    """
+    Given a config file pointing to valid test data, and an alias file for school ids for RRS200 -> RRS100
+    When the extract command is run, with an output to a temporary directory
+    Then a masterfile should be written with the school ID replaced
+    """
+    # Arrange
+    data_path = "tests/data"
+    test_config = {
+        "2021": {
+            "dispatch_list": f"{data_path}/dispatch_list.xlsx",
+            "current_year": {
+                "coded_data_file": f"{data_path}/redcap/extract.csv",
+                "label_data_file": f"{data_path}/redcap/extract_labels.csv",
+            },
+            "previous_year": {
+                "coded_data_file": f"{data_path}/redcap/extract.csv",
+                "label_data_file": f"{data_path}/redcap/extract_labels.csv",
+            },
+        }
+    }
+
+    config_path = temp_out_dir / "config.toml"
+    with config_path.open("wb") as handle:
+        tomli_w.dump(test_config, handle)
+
+    alias_path = temp_out_dir / "alias.toml"
+    with alias_path.open("wb") as handle:
+        tomli_w.dump({"RRS200": "RRS100"}, handle)
+
+    # Act
+    extract(2021, config_file=config_path, output_dir=temp_out_dir)
```

Comment:
oh and I should have set this up to pass`alias_path` to the `school_aliases` argument in this call 🤦